lockhunter.json

```diff
@@ -0,0 +1,51 @@
+{
+    "homepage": "https://lockhunter.com/index.htm",
+    "version": "3.2.3",
+    "description": "a foolproof file unlocker",
+    "license": "Freeware",
+    "architecture": {
+        "64bit": {
+            "url": "https://lockhunter.com/exe/lockhuntersetup_3-2-3.exe",
+            "hash": "59e7051cb079be49ac2bb4dd44cbdb6d4e11b6466d779465c7d1a3ca59272663",
+            "pre_install": [
```

Comment:
Make that `pre_install` non architecture specific, since its same

Reply:
Or whats difference between `LockHunter,1` and `LockHunter,2`?

Reply:
Same as the 32-Bit version? Innounpack unpacks files named `LockHunter,1.exe` which is 64-bit and `LockHunter,2.exe` which is 32-Bit. The architecture specific `pre_install` renames the appropriate file and deletes the other.

Reply:
URL and hashes are same too. Make it only something like this:

```json
"url": "URL",
"hash": "Hash",
"architecture" : {
    "xbi": {
         "pre_install": "Rename-Item \"$dir\\LockHunter,X.exe\" \"$dir\\lockhunter.exe\""
    }
},
"post_install": "Remove-Item \"$dir\\LockHunter*,*.exe\""
```

Reply:
See the previous message... The two non-service exe files unpacked by `innoup` are of different architectures

Reply:
NO!

```json
{
    "homepage": "https://lockhunter.com/index.htm",
    "version": "3.2.3",
    "description": "a foolproof file unlocker",
    "license": "Freeware",
    "url": "https://lockhunter.com/exe/lockhuntersetup_3-2-3.exe",
    "hash": "59e7051cb079be49ac2bb4dd44cbdb6d4e11b6466d779465c7d1a3ca59272663",
    "architecture": {
        "64bit": {
            "pre_install": "Rename-Item \"$dir\\LockHunter,1.exe\" \"$dir\\lockhunter.exe\""
        },
        "32bit": {
            "pre_install": "Rename-Item \"$dir\\LockHunter,2.exe\" \"$dir\\lockhunter.exe\""
        }
    },
    "post_install": "Remove-Item \"$dir\\LockHunter*,*.exe\"",
    "innosetup": true,
    "checkver": {
        "url": "https://lockhunter.com/download.htm",
        "re": "Version: ([\\d\\.]+)"
    },
    "bin": "lockhunter.exe",
    "shortcuts": [
        [
            "lockhunter.exe",
            "LockHunter"
        ]
    ],
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://lockhunter.com/exe/lockhuntersetup_$dashVersion.exe"
            },
            "32bit": {
                "url": "https://lockhunter.com/exe/lockhuntersetup_$dashVersion.exe"
            }
        }
    },
}

```